### PR TITLE
Disambiguation Harry Potter

### PR DESCRIPTION
**Pull request by MaathavanJkr**

### CODE REVIEW

File: texts/formatted_harry_potter.txt [hunk 1]
```diff
@@ -0,0 +1,1679 @@
+"<Harry>"
+	"Harry" np ant m sg
+"<Potter>"
+	"potter" n sg
+;	"Potter" np cog sg
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: Potter is his surname so it's a proper noun

File: texts/formatted_harry_potter.txt [hunk 2]
```diff
@@ -0,0 +1,1679 @@
...
+"<is>"
+	"be" vbser pres p3 sg
+"<a>"
+	"a" det ind sg
+"<series>"
+;	"series" n sg
+	"series" n pl
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This should be a singular noun

File: texts/formatted_harry_potter.txt [hunk 3]
```diff
@@ -0,0 +1,1679 @@
...
+"<of>"
+	"of" pr
+"<fantasy>"
+	"fantasy" n sg
+"<novels>"
+	"novel" n pl
+"<written>"
+;	"write" vblex pp
+	"written" adj
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This is a present participle

File: texts/formatted_harry_potter.txt [hunk 4]
```diff
@@ -0,0 +1,1679 @@
...
+"<by>"
+;	"by" adv
+	"by" pr
+"<British>"
+;	"British" adj
+;	"British" n sg
+	"British" n pl
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: "British" describes "author" so it's an adjective

File: texts/formatted_harry_potter.txt [hunk 5]
```diff
@@ -0,0 +1,1679 @@
...
+"<author>"
+;	"author" n sg
+	"author" vblex inf
+	"author" vblex pres
+	"author" vblex imp
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This hasn't been fully disambiguated and also it's a noun

**Comment by MaathavanJkr** [comment]
@dolphingarlic Have a look now. :)

File: texts/formatted_harry_potter.txt [hunk 6]
```diff
@@ -0,0 +1,1679 @@
...
+	"Harry" np ant m sg
+"<'s>"
+	"'s" gen
+;	"be" vbser pres p3 sg
+;	"have" vbhaver pres p3 sg
+"<struggle>"
+;	"struggle" n sg
+	"struggle" vblex inf
+;	"struggle" vblex pres
+;	"struggle" vblex imp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: follows a possessive, so it should be a noun

File: texts/formatted_harry_potter.txt [hunk 7]
```diff
@@ -0,0 +1,1679 @@
...
+	"Ron" np ant m sg
+"<Weasley>"
+	"Weasley" np cog sg
+"<,>"
+	"," cm
+"<all>"
+;	"all" adv
+;	"all" predet sp
+	"all" det qnt pl
+;	"all" prn qnt mf sp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: I think this should probably be pronoun (see tagging guidelines for "several" and "each").

File: texts/formatted_harry_potter.txt [hunk 8]
```diff
@@ -0,0 +1,1679 @@
...
+;	"have" vbhaver pres p3 sg
+"<struggle>"
+;	"struggle" n sg
+	"struggle" vblex inf
+;	"struggle" vblex pres
+;	"struggle" vblex imp
+"<against>"
+	"against" adj
+	"against" adv
+	"against" pr
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: not disambiguated

File: texts/formatted_harry_potter.txt [hunk 9]
```diff
@@ -0,0 +1,1679 @@
...
+;	"since" adv
+;	"since" cnjsub
+	"since" pr
+"<the>"
+	"the" det def sp
+"<release>"
+;	"release" n sg
+	"release" vblex inf
+;	"release" vblex pres
+;	"release" vblex imp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: after determiner and before preposition, should be noun

File: texts/formatted_harry_potter.txt [hunk 10]
```diff
@@ -0,0 +1,1679 @@
...
+"<popularity>"
+	"popularity" n sg
+"<,>"
+	"," cm
+"<critical>"
+	"critical" adj
+"<acclaim>"
+	"acclaim" vblex inf
+;	"acclaim" vblex pres
+;	"acclaim" vblex imp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: should be a noun, I think, though that isn't listed

File: texts/formatted_harry_potter.txt [hunk 11]
```diff
@@ -0,0 +1,1679 @@
...
+"<attracted>"
+	"attract" vblex pp
+;	"attract" vblex past
+"<a>"
+	"a" det ind sg
+"<wide>"
+	"wide" adj sint
+"<adult>"
+;	"adult" adj
+	"adult" n sg
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: modifies a noun -> adjective

File: texts/formatted_harry_potter.txt (hunk 10 above)
Comment (MaathavanJkr) [reply texts/formatted_harry_potter.txt]: Yeah i leaved as it not found

File: texts/formatted_harry_potter.txt (hunk 9 above)
Comment (MaathavanJkr) [reply texts/formatted_harry_potter.txt]: Oops these both i noticed as Nouns. But missed.

File: texts/formatted_harry_potter.txt (hunk 11 above)
Comment (MaathavanJkr) [reply texts/formatted_harry_potter.txt]: Yup i missed this. :)

File: texts/formatted_harry_potter.txt (hunk 10 above)
Comment (MaathavanJkr) [reply texts/formatted_harry_potter.txt]: What to do should i have to add that?

Comment (mr-martian) [reply texts/formatted_harry_potter.txt]: I would add it to the list here

File: texts/formatted_harry_potter.txt [hunk 12]
```diff
@@ -0,0 +1,1679 @@
...
+"<literature>"
+	"literature" n sg
+"<.>"
+	"." sent
+"<As>"
+;	"as" adv
+;	"as" preadv
+;	"as" cnjsub
+;	"as" cnjadv
+	"as" pr
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: the tagging guidelines don't address this, but 2 prepositions in a row seems a little odd to me - might want to ask about this on IRC

File: texts/formatted_harry_potter.txt [hunk 13]
```diff
@@ -0,0 +1,1679 @@
...
+;	"book" vblex pres p3 sg
+"<consecutively>"
+	"consecutively" adv
+"<set>"
+;	"set" n sg
+	"set" vblex inf
+;	"set" vblex pres
+;	"set" vblex past
+;	"set" vblex pp
+;	"set" vblex imp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: not infinitive

File: texts/formatted_harry_potter.txt [hunk 14]
```diff
@@ -0,0 +1,1679 @@
...
+"<records>"
+;	"record" n pl
+	"record" vblex pres p3 sg
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: noun

File: texts/formatted_harry_potter.txt [hunk 15]
```diff
@@ -0,0 +1,1679 @@
...
+"<the>"
+	"the" det def sp
+"<fastest>"
+	"fast" adj sint sup
+"<->"
+	"-" guio
+"<selling books>"
+	"sell# books" vblex pprs
+;	"sell# books" vblex subs
+;	"sell# books" vblex ger
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: functions as a noun, so should be `<ger>`

File: texts/formatted_harry_potter.txt [hunk 16]
```diff
@@ -0,0 +1,1679 @@
...
+"<Potter>"
+;	"potter" n sg
+	"Potter" np cog sg
+"<and>"
+	"and" cnjcoo
+"<the>"
+	"the" det def sp
+"<Cursed>"
+	"curse" vblex past
+;	"curse" vblex pp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: participle

File: texts/formatted_harry_potter.txt [hunk 17]
```diff
@@ -0,0 +1,1679 @@
...
+"<Child>"
+	"child" n sg
+"<,>"
+	"," cm
+"<based>"
+;	"based" adj
+	"base" vblex past
+;	"base" vblex pp
```
Comment (mr-martian) [inline texts/formatted_harry_potter.txt]: also participle

File: texts/formatted_harry_potter.txt [hunk 18]
```diff
@@ -0,0 +1,1681 @@
...
+;	"script" vblex inf
+;	"script" vblex pres
+;	"script" vblex imp
+"<was>"
+;	"be" vbser past p1 sg
+	"be" vbser past p3 sg
+"<published>"
+;	"publish" vblex pp
+	"publish" vblex past
+;	"published" adj
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This is a past participle

File: texts/formatted_harry_potter.txt [hunk 19]
```diff
@@ -0,0 +1,1681 @@
...
+"<seven>"
+	"seven" num pl
+"<books>"
+	"book" n pl
+;	"book" vblex pres p3 sg
+"<were>"
+	"be" vbser past
+"<adapted>"
+;	"adapt" vblex pp
+	"adapt" vblex past
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: Past participle

File: texts/formatted_harry_potter.txt [hunk 20]
```diff
@@ -0,0 +1,1681 @@
...
+;	"which" prn itg mf sp
+	"which" prn rel an mf sp
+"<is>"
+	"be" vbser pres p3 sg
+"<the>"
+	"the" det def sp
+"<third>"
+	"third" num sg
+;	"third" n sg
+;	"third" adj ord
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This is an adjective

File: texts/formatted_harry_potter.txt [hunk 21]
```diff
@@ -0,0 +1,1681 @@
...
+	"Potter" np cog sg
+"<franchise>"
+	"franchise" n sg
+"<was>"
+;	"be" vbser past p1 sg
+	"be" vbser past p3 sg
+"<estimated>"
+;	"estimated" adj
+	"estimate" vblex past
+;	"estimate" vblex pp
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This is a past participle

File: texts/formatted_harry_potter.txt [hunk 22]
```diff
@@ -0,0 +1,1681 @@
...
+"<A>"
+	"a" det ind sg
+"<series>"
+	"series" n sg
+;	"series" n pl
+"<of>"
+	"of" pr
+"<many>"
+;	"many" det qnt pl
+	"many" prn ind mf pl
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This is a determiner

File: texts/formatted_harry_potter.txt [hunk 23]
```diff
@@ -0,0 +1,1681 @@
...
+"<genres>"
+	"genre" n pl
+"<,>"
+	"," cm
+"<including>"
+;	"including" adv
+	"including" pr
+;	"include" vblex pprs
+;	"include" vblex subs
+;	"include" vblex ger
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: This isn't a preposition

File: texts/formatted_harry_potter.txt [hunk 24]
```diff
@@ -0,0 +1,1681 @@
...
+	"a" det ind sg
+"<pentalogy>"
+	"*pentalogy"
+"<of>"
+	"of" pr
+"<spin>"
+	"spin" n sg
+;	"spin" vblex inf
+;	"spin" vblex pres
+;	"spin" vblex imp
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: I think this is a verb but I'm not sure

File: texts/formatted_harry_potter.txt [hunk 25]
```diff
@@ -0,0 +1,1681 @@
...
+"<insight>"
+	"insight" n sg
+"<,>"
+	"," cm
+"<and>"
+	"and" cnjcoo
+"<a>"
+	"a" det ind sg
+"<pentalogy>"
+	"*pentalogy"
```
Comment (dolphingarlic) [inline texts/formatted_harry_potter.txt]: Please add analyses to unknown words like this

**Pull request merged**